src/translations/en.json

```diff
@@ -18,6 +20,7 @@
   "deleteStudyMsg": "Are you sure you want to delete the study?",
   "studyNameDidNotMatchMsg": "The given study name did not match with ",
   "studyDeletedSuccessMsg": "Study deleted",
+  "deleteStudyError": "Cannot remove the study if you are not the owner",
```

Comment:
There can be other sources of errors when deleting a study, but maybe we can do this in a later PR